address_book_ui/src/app/components/contact-block/contact-block.component.scss

```diff
@@ -0,0 +1,21 @@
+.contact-info {
+    padding: 10px 0;
```

Comment:
* we use the 8 point grid system in UX - 'the principle of 8pt Grid is that use multiples of 8 (8, 16, 24, 32, 40, 48, 56, etc.) to layout, dimensions, padding, and margin of elements'
* and in this case you can easily set rem instead of px